nautilus_trader/adapters/dydx/websocket/client.py

```diff
@@ -147,10 +157,26 @@ async def connect(self) -> None:
             self._reconnect_task = self._loop.create_task(self._reconnect_guard())
 
     def _msg_handler(self, raw: bytes) -> None:
+        """
+        Handle pushed websocket messages.
+
+        Parameters
+        ----------
+        raw : bytes
+            The received message in bytes.
+        """
```

Comment:
Small point, but we tend to leave a single blank line at the end of the docstring. The `docformatter` lint may just handle this when its turned back on.

Reply:
Thanks for formatting the code already.